src/sentry/grouptype/grouptype.py

```diff
@@ -112,6 +112,7 @@ class PerformanceUncompressedAssetsGroupType(GroupType):
     slug = "performance_uncompressed_assets"
     description = "Uncompressed Asset"
     category = GroupCategory.PERFORMANCE.value
+    ignore_limit = 10
```

Comment:
Is this in use already? I think you have to also update `DEFAULT_GROUPHASH_IGNORE_LIMIT` in `event_manager.py`, because that code is still using the constant, not the new dataclass

Reply:
Interesting

Reply:
😅 I am in the middle of a refactor to use the new GroupType dataclass so you'd only change / set the `ignore_limit` sorry it's wonky for a little bit while I move things over.

Reply:
The two configs caught me by surprise, I'd probably keep the un-used one out of main branch until you are ready to switch them next time? But all good 👍